cdap-docs/_common/common-build.sh

```diff
@@ -140,25 +126,39 @@ function usage() {
   echo "    license-pdfs   Clean build of License Dependency PDFs"
   echo
   echo "    check-includes Check if included files have changed from source"
-  echo "    depends        Build Site listing dependencies"
-  echo "    sdk            Build SDK"
   echo "  with"
   echo "    source         Path to $PROJECT source for javadocs, if not $PROJECT_PATH"
-  echo "    test_includes  local, remote or neither (default: remote); must specify source if used"
   echo
 }
 
+function echo_red_bold() {
+  if [[ "${OSTYPE}" == "darwin"* ]]; then
+    echo -e "${RED}${BOLD}${1}${NC}${2}"
+  else
+    echo "${1}${2}"
+  fi
+}
```

Comment:
Fixed. Color is now set by environment variable rather than OS, and no longer uses the '-e' flag in echo statements.